ios/FluentUI/People Picker/Presence.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+// MARK: - Presence
+
+@objc(MSFPresenceSize)
+public enum PresenceSize: Int {
+    case normal
+    case large
+    case extraLarge
+
+    public var sizeValue: CGFloat {
+        switch self {
+        case .normal:
+            return 10
+        case .large:
+            return 12
+        case .extraLarge:
+            return 16
+        }
+    }
+}
+
+@objc(MSFPresence)
+public enum Presence: Int, CaseIterable {
+    case none
+    case available
+    case away
+    case busy
+    case doNotDisturb
+    case outOfOffice
+    case offline
+    case unknown
+    case blocked
+
+    public func image(size: PresenceSize) -> UIImage? {
```

Comment:
accessibility label?

Reply:
The accessibility label in Teams when status is enabled is heavily customized depending on the context.
Let's provide a default accessibility label but allow clients to customize it.